### PR TITLE
jmeter 5.1.1

### PR DESCRIPTION
**Pull request by plokijuma**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
Thanks @plokijuma

File: Formula/jmeter.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Jmeter < Formula
   desc "Load testing and performance measurement application"
   homepage "https://jmeter.apache.org/"
-  url "https://www.apache.org/dist/jmeter/binaries/apache-jmeter-5.1.1.tgz"
+  url "https://www.apache.org/dyn/closer.cgi?path=jmeter/binaries/apache-jmeter-5.1.1.tgz"
```
Comment (zbeekman) [inline Formula/jmeter.rb]: @fxcoudert This needs to be reverted. The link you provided does not download a tarball.

**Review by zbeekman — COMMENTED** [review]
Need to revert @fxcoudert's commit

**Comment by zbeekman** [comment]
@fxcoudert please accept my apologies. I'm a liar and my pants are very much on fire.

**Pull request closed without merging**